src/output-json-http.c

```diff
@@ -202,6 +202,13 @@ static void JsonHttpLogJSONBasic(json_t *js, htp_tx_t *tx)
         }
     }
 
+    /* port */
+    if (tx->request_port_number >= 0)
```

Comment:
is this only set if different? If not, does it make sense to compare it against the flow dest port and only log it if different?

Reply:
not necessarily, this is set only if the hostname header has a port in it, or if the url has it. The idea behind the patch is to log an information that otherwise will be lost. I can foresee several cases where tcp dest  port != http port but request is "normal", ex http proxy. I'm not sure whether we are ok to lose the information "hostname has a port" because the tcp port coincide with this value.